pkg/manager/member/tidb_member_manager.go

```diff
@@ -393,12 +393,24 @@ func getTiDBConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	}
 
 	plugins := tc.Spec.TiDB.Plugins
-	startScript, err := RenderTiDBStartScript(&TidbStartScriptModel{
-		ClusterName:     tc.Name,
+	tidbStartScriptModel := &TidbStartScriptModel{
 		EnablePlugin:    len(plugins) > 0,
 		PluginDirectory: "/plugins",
 		PluginList:      strings.Join(plugins, ","),
-	})
+	}
+
+	if len(tc.Spec.PDAddress) > 0 && tc.Spec.PD == nil {
+		pdAddress := ""
+		for _, address := range tc.Spec.PDAddress {
+			removeHttpAddress := strings.ReplaceAll(address, "http://", "")
+			removeHttpsAddress := strings.ReplaceAll(removeHttpAddress, "https://", "")
+			pdAddress += removeHttpsAddress + ","
+		}
+		tidbStartScriptModel.Path = strings.TrimSuffix(pdAddress, ",")
+	} else {
+		tidbStartScriptModel.Path = controller.PDMemberName(tc.Name) + ":2379"
```

Comment:
Will this trigger the rolling update of the TiDB since the start script in the configmap is changed?